src/main/java/com/powsybl/openloadflow/dc/equations/DcEquationSystemUpdater.java

```diff
@@ -45,6 +45,7 @@ public void onDisableChange(LfElement element, boolean disabled) {
                 break;
             case BRANCH:
             case HVDC:
+            case SHUNT_COMPENSATOR:
```

Comment:
@obrix can you remove this fix? We have to open a dedicated PR that shows the bug. Can you open it in parallel and I am going to do a use case. Thanks !

Reply:
Okay, I just opened a separate pull request.